build/StablePackagesInfo.props

```diff
@@ -0,0 +1,5 @@
+<Project>
+    <PropertyGroup>
+        <StableProjects>Microsoft.ML;Microsoft.ML.CpuMath;Microsoft.ML.ImageAnalytics;Microsoft.ML.LightGBM;Microsoft.ML.Mkl.Components;Microsoft.ML.Mkl.Redist</StableProjects>
```

Comment:
it's `LightGbm` after recent changes (casing of GBM got change)

Reply:
Since creation of Issue we also created Microsoft.ML.FastTree package which I believe should be part of stable packages.